### ISSUE TITLE
Switch Freezes while in dock mode when in sleep mode for too long

### ISSUE THREAD

**Issue by GyanStriker** [cathery/sys-con#307]
**Description**
As the title says Switch Freezes while in dock mode when in sleep mode for too long, i know other people have reported this problem but does anyone have a solution as of 11/18/2022 ???

**How to reproduce**
From my testing, while sys-con is enabled if you let your console on sleep mode from about: 0 - 15 minutes, you can turn on your switch, if the consoles stays in sleep mode from 20 - 30 minutes, the switch freezes in a black screen and you cant turn on your nintendo switch. [Only solution is to press the power button for 15 seconds to force it to turn off , then press it again to turn it on]

**Environment**
I want to know if there is a solution for my current switch version

- Switch Firmware version: 15.0.1
- Version of Atmosphere: 1.4.0
- Latest version of sys-con at the moment: 0.6.4
- The only other sysmodule i have is: sys-ftpd-light 1.0.4

**Comment by VelocityRa** [comment]
Do you need to have any controllers connected for this to happen? 

For example #248 mentions a DS4 specifically though not sure if that's required.

**Comment by GyanStriker** [comment]
> Do you need to have any controllers connected for this to happen?
> 
> For example #248 mentions a DS4 specifically though not sure if that's required.

No, this error happens if there is a controller connected or not, it happens both ways

**Comment by o0Zz** [comment]
Try this fork -> https://github.com/o0Zz/sys-con/releases
